generate-spec

```diff
@@ -523,13 +525,26 @@ foreach ($parsedRoutes as $key => $value) {
 			continue;
 		}
 
+		$operationId = [
+			$tagName,
```

Comment:
this uses the class name (default behaviour) tags only, not the tag name given within the scope, see `$routeTags[$scope] ?? [$tagName]` below?

Reply:
Yes. The variable name is not really well chosen, but I didn't want to refactor it here.
We have to use the combination of controller name, method name and postfix to ensure operationIds are unique.
The tags are only used to group operations, but shouldn't have been used to generate the operationId.

Reply:
But now it will "break" existing clients build on top of this?

Reply:
It is a breaking change if you generated code for this and publish it as a library, but the API itself is not breaking since all the same operations still exist.